bootstrap_functions.sh

```diff
@@ -26,6 +26,9 @@ function install_utils() {
         tree colordiff libxml2-utils xmlstarlet nmap
 
     chown -R vagrant.vagrant /home/vagrant
+
+    cat /vagrant/files/bashrc >> /root/.bashrc
+    cat /vagrant/files/bashrc >> /home/vagrant/.bashrc
```

Comment:
I have not tried spinning up a new VM for this yet, but from my existing redcap_deployment these aliases are only functional as root

Reply:
Because the .my.cnf for the vagrant user is chronically deleted by the deployment tools. I feel like that's a different bug that we might or might not address.

Reply:
@ChemiKyle, are you OK with this PR?